### DISCUSSION TITLE
월드 내 좌상단 버튼 숨기기

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#330] [5. Q&A]
안녕하세요.
유니티로 제작한 월드에서 좌상단에 보여지는 버튼을 숨길 수 있는 방법이 있나요?
가이드 부탁드립니다.
감사합니다.

![image](https://user-images.githubusercontent.com/5335151/152689256-1592884f-0813-4eeb-86c7-b61b5fbe9cb4.png)

**Comment by moondory77-naverz** [comment]
안녕하세요. 
말씀주신 UI의 경우, ZEPETO World의 공통 UI로 
현재 숨기기 옵션은 따로 제공하고 있지 않습니다.

감사합니다.
